modules/standard/Math.chpl

```diff
@@ -1195,7 +1195,15 @@ module Math {
     return a;
   }
 
+  /* Returns true if `x` and `y` are approximately equal, else returns false. */
+  inline proc isclose(x : real(64), y : real(64), rtol : real(64), atol : real(64), either : bool): bool {
```

Comment:
Few things here:

- Chapel supports generic arguments to functions. What you have here would only work for 64-bit floating points. But I think it'd be nice to support any `int` and `real` type (regardless of their bit width)
- As a user, I'd definitely want some defaults for `rtol` and `atol` and not worry about them for most of the time
- `either` sounds wrong to me looking at its behavior. It should be named `both` or something along those lines
- And this last boolean argument should also have a default value. (My instinct says we should be checking only one of the tolerances by default)

Reply:
We'll also need to check whether `rtol` and `atol` are reasonable values (e.g. neither can be negative)

Reply:
I agree with the feedback so far, but note that some design decisions are still converging over in https://github.com/chapel-lang/chapel/issues/13898, e.g. dropping `either` arg, so there will probably be some additional changes beyond this.